source/algorithms.tex

```diff
@@ -610,6 +598,31 @@
 #include <initializer_list>
 
 namespace std {
+  namespace ranges {
+
```

Comment:
This empty line seems unconventional; we usually have content on the first line of a namespace.

Reply:
That sounds right; @jensmaurer  please delete the blank line?

Reply:
Oh never mind, I can edit this directly; added a fixup commit.